### PR TITLE
Touch up config stuff a bit

### PR DESCRIPTION
**Pull request by polyzen**

### CODE REVIEW

File: config [hunk 1]
```diff
@@ -45,10 +45,10 @@ background = #3f3f3f
...
 
-# if unset, will reverse foreground and background
+# If unset, will reverse foreground and background
 highlight = #2f2f2f
 
-# colors from color0 to color254 can be set
+# Colors from color0 to color254 can be set
 color0 = #3f3f3f
 color1 = #705050
 color2 = #60b48a
```
Comment (jelly) [inline config]: This looks fine to me.

File: config [hunk 2]
```diff
@@ -38,9 +38,9 @@ filter_unmatched_urls = true
...
 
-foreground = #dcdccc
-foreground_bold = #ffffff
-background = #3f3f3f
+#foreground = #dcdccc
+#foreground_bold = #ffffff
+#background = #3f3f3f
 
 # 20% background transparency (requires a compositor)
 #background = rgba(63, 63, 63, 0.8)
```
Comment (jelly) [inline config]: Sure this is fine too.

File: config [hunk 3]
```diff
@@ -8,6 +8,7 @@ dynamic_title = true
 urgent_on_bell = true
 clickable_url = true
 font = Monospace 9
+#fullscreen = true
 scrollback_lines = 10000
 search_wrap = true
 #icon_name = terminal
```
Comment (jelly) [inline config]: Does not seem to work, the config option. Might be an i3 thing, but F11 works for me. So this is something I'll need to digg in.

Apart from that issue it looks fine to me.

Comment (polyzen) [inline config]: Setting `fullscreen = false` appears to do its job of disabling F11.

**Comment by jelly** [comment]
I just realized, that this commit might break the color conifguration of a lot of users. Which I'd say would be annoying if it's so

**Comment by jelly** [comment]
So to conclude this. Please make a separate PR for:

config add fullscreen option df93649cc9590d2cf22316c4a8434dedac86cf5b

capatilize letters cosnsitently 8eea0ffa8c2c2b203ba0e652336dac3a3e953ac2

comment out deafult options dc5ec76830161e05b3781c4b8aabbde8a2a72d5a

Then I'll merge those and we can take a deeper dive into the alphabetic sorting.

**Comment by polyzen** [comment]
> this commit might break

Which?

**Comment by polyzen** [comment]
Left 5b7f7df (config: Sort contents of [options] alphabetically) as it depends on the previous commits, and I would be amazed if it causes any issue.

**Pull request merged**